suites/squid/rgw/sanity_rgw_multisite.yaml

```diff
@@ -625,3 +625,13 @@ tests:
           config:
             script-name: test_bucket_request_payer.py
             config-file-name: test_bucket_request_payer.yaml
+  - test:
```

Comment:
@viduship 
I think this suite is only present under stage run which will not run until the end of the release testing..
https://gitlab.cee.redhat.com/rhcs-qe/cephqe-jenkins/-/blob/main/metadata/squid/rgw.yaml#L594

could you please check and move this test to some other weekly or regression run suites..

Reply:
done @hmaheswa